ui/src/app/services/accounts/doctors.service.ts

```diff
@@ -10,4 +15,23 @@ export class DoctorsService extends DataService {
   constructor(http: Http) {
     super('/doctors', http);
    }
+
+   addDoctor(doctor: { email: string, password: string, firstName: string, lastName:string }) {
+  //   let newDoctor = {
+  //     id: 1,
+  //     email: doctor.email,
+  //     firstName: doctor.firstName,
+  //     lastName: doctor.lastName
+  // };
+
```

Comment:
Let's remove unnecessary comments. Since everything is working as desired now, I believe we don't need to keep this in code anymore. 

Correct me if I'm wrong or remove comments, please.